packages/math-input/src/components/input/math-input.tsx

```diff
@@ -623,6 +624,13 @@ class MathInput extends React.Component<Props, State> {
         if (!this.state.focused) {
             this.focus();
         }
+
+        // If the user clicked on the input using a mouse or tap gesture,
+        // we want to set the focus to the inputRef so that the keyUp
+        // event will be triggered when the user types on the keyboard.
+        // This is necessary to support Chromebooks as they use mobile user
+        // agents, do not simulate touch events, and have physical keyboards.
+        this.inputRef?.focus();
```

Comment:
For both of these fixes, immediately above them we do the check on `this.state.focused` and then call `this.focus()`. 

I'm wondering if these `inputRef?.focus()` calls might belong in there?  Of if the state.focused is false in some cases? 

I trust you've tested this, but just had the question in case you knew. 

I'll approve it so you aren't blocked and leave it to you if you stick with this or integrate into the `this.focus()` function.

Reply:
Yes! Unfortunately state.focused might be false in some cases, and we want to focus the input ref every time the user clicks or taps on the input. 

I have an upcoming PR that should help clean up things slightly, but I had to revert those commits as they were causing a flumoxing test failure that we haven't been able to nail down yet. I've also been chatting with design about different approaches to focus handling (perhaps a button like desktop), or @nishasy mentioned the possibility of having Chromebooks be recognized as "desktop" machines which would let us cut out a lot of the edge-cases we're having to handle specifically for Chromebooks. 

I'll land this PR as is for now, but I definitely want to clean up the focus function in the future.